### PR TITLE
settings: smoother translate strings (fixes #4302)

### LINKED ISSUE #4302: translate text in settings
ensure texts in this page are translated to all languages
Screenshot_20240830_125450

### PR DESCRIPTION
**Pull request by Kuljeet1998**
Fixes #4302 

https://github.com/user-attachments/assets/d1e83585-f025-49a4-b1d2-38e386f4bf01

### CODE REVIEW

**Comment by Okuro3499** [comment]
@Kuljeet1998 resolve the conflits

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -1119,6 +1119,8 @@
     <string name="show_more">show more</string>
     <string name="server_address_content_description">server address: %1$s</string>
     <string name="member_only_allowed">join as a member to access this feature</string>
+    <string name="select_theme_mode">select theme mode</string>
+    <string name="beta_resources_auto_download">Beta resources auto download</string>
     <string name="select_theme_mode">select theme mode</string>]
```
Comment (deeppp15) [inline app/src/main/res/values/strings.xml:1124]: Hi @Kuljeet1998  select_theme_mode string id already present

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -1119,7 +1119,6 @@
     <string name="show_more">show more</string>
     <string name="server_address_content_description">server address: %1$s</string>
     <string name="member_only_allowed">join as a member to access this feature</string>
-    <string name="select_theme_mode">select theme mode</string>
     <string name="beta_resources_auto_download">Beta resources auto download</string>
     <string name="select_theme_mode">select theme mode</string>]
```
Comment (deeppp15) [inline app/src/main/res/values/strings.xml:1123]: Hi @Kuljeet1998  ] bracket present at the end of the select_theme_mode

Comment (Okuro3499) [reply app/src/main/res/values/strings.xml:1123]: I don't see the bracket but it should not be there

Comment (Kuljeet1998) [reply app/src/main/res/values/strings.xml]: Okay I have removed the bracket - please check @Okuro3499

**Review by Okuro3499 — APPROVED** [review]

**Review by AdhamElAsfar99 — APPROVED** [review]

**Review by deeppp15 — APPROVED** [review]

**Review by dogi — APPROVED** [review]

**Review by dogi — APPROVED** [review]

**Pull request merged**